index.html

```diff
@@ -31,6 +31,12 @@ <h2>日本で近日開催のイベント</h2>
   <!-- <div class="span12" style="padding-bottom: 2em;">
     coming soon..
   </div> -->
+  <a href="https://railsgirls.com/sapporo-2nd.html" class="span4 event" style="background: url(images/railsgirls-sq.png) 0px -70px / 100% no-repeat;">
```

Comment:
```suggestion
  <a href="https://railsgirls.com/sapporo-2nd.html" class="span4 event" style="background: url(/images/railsgirls-sq.png) 0px -70px / 100% no-repeat;">
```

画像は表示されているのですが、合わせて `/images/` としておいた方がよさそうです！